data/json/recipes/food/dairy_products.json

```diff
@@ -127,6 +127,53 @@
       [ [ "bell_pepper", 1 ], [ "tomato", 1 ], [ "cucumber", 1 ] ]
     ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "NO_EXERCISE",
+    "result": "mozzarella_cheese",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 1,
+    "time": "35 m",
+    "charges": 8,
+    "book_learn": [ [ "dairy_book", 1 ] ],
+    "qualities": [ { "id": "COOK", "level": 1 } ],
+    "proficiencies": [
+      { "proficiency": "prof_food_prep" },
+      { "proficiency": "prof_cheesemaking_1" },
+      { "proficiency": "prof_cheesemaking_2" }
+    ],
+    "tools": [ [ [ "water_boiling_heat", 2, "LIST" ] ], [ [ "colander_steel", -1 ] ], [ [ "cotton_patchwork", -1 ] ] ],
+    "components": [
+      [ [ "milk", 5 ], [ "milk_raw", 5 ] ],
+      [ [ "vinegar", 1 ], [ "chem_citric_acid", 1 ] ],
+      [ [ "salt", 1 ] ],
+      [ [ "water", 1 ], [ "water_clean", 1 ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "NO_EXERCISE",
+    "result": "fried_mozzarella_sticks",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 1,
+    "time": "25 m",
+    "charges": 8,
+    "book_learn": [ [ "dairy_book", 1 ] ],
+    "qualities": [ { "id": "COOK", "level": 1 } ],
+    "tools": [ [ [ "water_boiling_heat", 2, "LIST" ] ] ],
```

Comment:
```suggestion
    "tools": [ [ [ "water_boiling_heat", 2, "LIST" ] ], [ [ "frying_oil", 1, "LIST" ] ] ],
```

Reply:
Thank you for the fixes while I was offline, hopefully that cleared it all up too? I wasn't aware I could set oil as a tool instead of ingredient.

Reply:
Yeah otherwise all of the cooking oil calories go into the resulting food item. :)